patchwork-events-input/src/main/java/net/patchworkmc/mixin/event/input/MixinMouse.java

```diff
@@ -32,36 +28,26 @@
 
 import net.minecraft.client.Mouse;
 
-@Mixin(Mouse.class)
-public abstract class MixinMouse {
-	@Shadow
-	boolean middleButtonClicked;
-	@Shadow
-	abstract boolean wasLeftButtonClicked();
-	@Shadow
-	abstract boolean wasRightButtonClicked();
-	@Shadow
-	abstract double getX();
-	@Shadow
-	abstract double getY();
+import net.patchworkmc.api.input.ForgeMouse;
+import net.patchworkmc.impl.event.input.InputEvents;
 
+@Mixin(Mouse.class)
+public abstract class MixinMouse implements ForgeMouse {
```

Comment:
This doesn't actually implement the interface?

Reply:
huh. whoops. i must've gotten it in place and then forgot to actually implement it.